src/esp/assets/ResourceManager.cpp

```diff
@@ -1369,25 +1362,10 @@ gfx::PbrMaterialData::uptr ResourceManager::buildPbrShadedMaterialData(
     finalMaterial->metallicTexture =
         textures_[textureBaseIndex + material.metalnessTexture()].get();
     if (!material.hasAttribute(Mn::Trade::MaterialAttribute::Metalness)) {
-      finalMaterial->metallic = 1.0f;
+      finalMaterial->metallic = 1.0f;  // based on GlTF 2.0 Spec
     }
   }
```

Comment:
To clarify:
I did not "invent" this `PbrMaterialData`. I just followed the `PhongMaterialData` which was there before, and created it. 
I am not against refactoring the code so that we can migrate the code to adopt the Magnum built-in `Trade::MaterialData`.

Reply:
This is DONE.

Reply:
Come on, it's not :sweat_smile: You need to use the `hasAttribute()` / `has*Texture()` only for textures, because there's no "default texture ID" that could be used. For everything else you can just use the `metalness()` etc. getter. Let me repeat once again -- instead of:

```cpp
  if (material.hasAttribute(Mn::Trade::MaterialAttribute::Metalness)) {
    finalMaterial->metallic = material.metalness();
  }
```

it can be just:

```cpp
  finalMaterial->metallic = material.metalness();
```

And the same for, as I said above:

- base color
- normal texture scale
- emissive color
- roughness

While we're deleting code, let's do that completely :wink:

Reply:
OK. Make sense. Will do.

Reply:
DONE.